src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java

```diff
@@ -143,6 +145,13 @@ public String uploadSourceFile(String blobName, long length) {
     return blobName;
   }
 
+  public String uploadFileWithContents(String blobName, String contents) {
+    var bytes = contents.getBytes(StandardCharsets.UTF_8);
+    var byteStream = new ByteArrayInputStream(bytes);
```

Comment:
Should put this in a try with resources so it closes.. Right now the stream never closes